src/browser_action/chrome.js

```diff
@@ -48,3 +48,12 @@ export function getOptions() {
     chrome.storage.sync.get({preferPhoneField: false}, resolve);
   });
 }
+
+export function getURL() {
+  return new Promise(resolve => {
+    chrome.tabs.query({active: true, lastFocusedWindow: true}, tabs => {
```

Comment:
FWIW all these methods return a promise, which generally results in simpler code when the callback isn't necessary

Reply:
Not sure what you mean here, but my promises knowledge is pretty damn poor. Can you explain?